### PR TITLE
Fix application disappearing from the taskbar when opening build configuration

### PR DESCRIPTION
**Pull request by somebirby**
Fix main window getting hidden when opening build configuration, making the app disappearing from the taskbar, creating a confusion situation for the user.

### CODE REVIEW

File: src/BuildConfigurator.h
Comment (N00byKing) [inline src/BuildConfigurator.h:1]: I think you might have added this file by mistake?
I'll review the other changes later

Comment (somebirby) [reply src/BuildConfigurator.h:1]: Indeed, woops! Fixed it.
Don't hesitate to be frank about this modification btw, it's my first pull request ever.

**Comment by somebirby** [comment]
Anything blocking you from merging this?

**Comment by N00byKing** [comment]
No, sorry, just forgot ^^'
Can you delete the code you commented? Then I can merge this :)

**Comment by somebirby** [comment]
Done

**Comment by N00byKing** [comment]
Thanks!

**Pull request merged**